nilearn/plotting/surf_plotting.py

```diff
@@ -312,6 +351,9 @@ def plot_surf_stat_map(surf_mesh, stat_map, bg_map=None,
         which is a name of a matplotlib colormap, or a matplotlib
         colormap object.
 
+    colorbar : bool, optional, default is False
+        If True, a colorbar is added.
```

Comment:
Could you please more details here as well as I commented above. These will be useful for users to know before.